### ISSUE TITLE
Project: FluentUI Beta Release

### ISSUE THREAD

**Issue by behowell** [microsoft/fluentui#19777]
# Project: FluentUI Beta Release

### Problem
We need to release FluentUI v9 beta, and someone needs to make sure that process stays on track and drives to completion.

### Appetite
4 weeks part-time. Ideally the release should happen by the end of September.

### Solution
* Track work needed to be done on https://github.com/microsoft/fluentui/projects/50
* Have weekly status meetings
* Triage/prioritize incoming work items to determine if it *must* happen before the beta is released.
    * Primarily this involves making sure that any major API changes happen before beta.

### Risks (Rabbit holes)
* Indecision about API decisions can delay the release
    * Need to drive towards decisions for RFCs, and make sure that PRs are reviewed promptly.
* This process could drag on if new work is continually added to the beta board.
    * Need to triage new items and only accept new work if it is truly blocking the beta release.

### Out of scope (No-gos)
* Anything that doesn't block the beta release.
    * Need to focus on the Priority 1 (Must) level items that need to happen before we release the first beta.
